feature-libs/cart/base/components/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -119,6 +120,7 @@ export class CartItemListComponent implements OnInit, OnDestroy {
         this.cartId = context.cartId;
       }
       if (context.items !== undefined) {
+        this._foreReRender = true;
```

Comment:
That means we re-render _all_ entries whenever they are provided.
Shouldn't we set forceReRender only if readOnly changes?

Reply:
i think you are right - it's sufficient to do if the read-only flag changes